src/components/tooltip/tooltip.tsx

```diff
@@ -40,21 +55,34 @@ export const Tooltip: FC<TooltipProps> = ({
   contentClassName,
   dynamicWidth,
   width,
+  minWidth = 120,
+  safeZone = 4,
+  zIndex = 9,
```

Comment:
I suggest to set the default value as `auto` to avoid unexpected behavior in new projects.

Reply:
![image](https://github.com/user-attachments/assets/99ebb8b0-e2ad-49c4-ac6a-ef6e7ee3202f)
'auto' in our project will not work in many places